pybaseball/team_batting.py

```diff
@@ -1,79 +1,36 @@
+import os
+
 import numpy as np
 import pandas as pd
-from bs4 import BeautifulSoup
 import requests
+from bs4 import BeautifulSoup
 
-def get_soup(start_season, end_season, league, ind):
-    url = "http://www.fangraphs.com/leaders.aspx?pos=all&stats=bat&lg={}&qual=0&type=8&season={}&month=0&season1={}&ind={}&team=0,ts&rost=0&age=0&filter=&players=0&page=1_100000"
-    url = url.format(league, end_season, start_season, ind)
-    s=requests.get(url).content
-    #print(s)
-    return BeautifulSoup(s, "lxml")
-
-def get_table(soup, ind):
-    table = soup.find('table', {'class': 'rgMasterTable'})
-    data = []
-    # couldn't find these in the table, hardcoding for now
-    if ind == 0:
-        headings = ["Team", "G","PA","HR","R","RBI","SB","BB%","K%","ISO","BABIP","AVG","OBP","SLG","wOBA","wRC+","BsR","Off","Def","WAR"]
-    else:
-        headings = ["Season","Team","G","PA","HR","R","RBI","SB","BB%","K%","ISO","BABIP","AVG","OBP","SLG","wOBA","wRC+","BsR","Off","Def","WAR"]
-
-    data.append(headings)
-    table_body = table.find('tbody')
-    rows = table_body.find_all('tr')
-    for row in rows:
-        cols = row.find_all('td')
-        cols = [ele.text.strip() for ele in cols]
-        data.append([ele for ele in cols[1:]])
-    data = pd.DataFrame(data)
-    data = data.rename(columns=data.iloc[0])
-    data = data.reindex(data.index.drop(0))
-    return data
-
-
-def postprocessing(data):
-    # fill missing values with NaN
-    data.replace(r'^\s*$', np.nan, regex=True, inplace = True)
-    data.replace(r'^null$', np.nan, regex=True, inplace = True)
-
-    # convert percent strings to float values
-    percentages = ['BB%', 'K%']
-    for col in percentages:
-        # skip if column is all NA (happens for some of the more obscure stats + in older seasons)
-        if data[col].count()>0:
-            data[col] = data[col].str.strip(' %')
-            data[col] = data[col].str.strip('%')
-            data[col] = data[col].astype(float)/100.
-        else:
-            pass
-
-    # convert columns to numeric
-    not_numeric = ['Team']
-    numeric_cols = [col for col in data.columns if col not in not_numeric]
-    data[numeric_cols] = data[numeric_cols].astype(float)
-    return data
-
```

Comment:
This is some sweet sweet dividends here

Reply:
I always love deleting code more than adding code.